pkgs/development/python-modules/pytest-subtests/default.nix

```
@@ -8,14 +8,14 @@
 
 buildPythonPackage rec {
   pname = "pytest-subtests";
-  version = "0.8.0";
+  version = "0.9.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.6";
```

Only Python [> 3.7](https://github.com/pytest-dev/pytest-subtests/blob/main/setup.py#L23) is supported.

```suggestion
  disabled = pythonOlder "3.7";
```